tests/test_cached_client.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python
+#
+# Copyright 2016 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+#
+# derived from https://github.com/verisign/python-confluent-schemaregistry.git
+#
+
+import sys
+import time
+
+from avro import schema
+
+if sys.version_info[0] < 3:
+    import mock_registry
+    import data_gen
+    import unittest
+else:
+    from tests import mock_registry
+    from tests import data_gen
+    import unittest2 as unittest
+
+from confluent_kafka.avro.cached_schema_registry_client import CachedSchemaRegistryClient
+from confluent_kafka.avro.serializer import util
+
+
+class TestCacheSchemaRegistryClient(unittest.TestCase):
+    def setUp(self):
+        # Make RecordSchema and PrimitiveSchema hashable
+        schema.RecordSchema.__hash__ = self.hash_func
```

Comment:
This seems to be getting duplicated a few different places. Any reason not to move it into, e.g., one of the `__init__.py` files that would guarantee it gets imported if you use anything from the `confluent_kafka.avro` module?